Testnet release plan.md

```diff
@@ -0,0 +1,55 @@
+# Testnet versions
+
+This is the large task breakdown I propose we add to the next versions of testnet.
+There will be bug fixes and UX improvements as we discover them, which will be treated as high priority.
+
+Note that there is no protocol work listed here. These are the tasks that are orthogonal to the protocol.
+
+
+## V2
+1. Events subscription
+2. Connect to a well-known Ethereum testnet
+   a) support restarting of nodes
+3. Improvements to Management contract
+   a) Support withdrawals
+4. Mempool improvements
+   a) Remove failed transactions
+
+
+## V3
+1. Support a limited number of user operated nodes
+   a) Catching up robustness
+   b) Script robustness
+   c) Instructions
+   d) Try outside of Azure?
+2. Design and implement reveleation period
+3. Finalise cryptography
+   a) key derivation (both the obscuro network key, and the symmetric rollup encryption keys) 
+   b) Calculate hashes on startup
+   c) Check that the rollup producer is attested
+   d) Add entropy to txs
+   e) Encrypt some errors returned from enclave
+4. Design and implement the final mempool
```

Comment:
I thought we didn't have a mempool! :-)

Reply:
we have a stone-age mempool now.
This would take it to the space age :)